content/review.rst

```diff
@@ -306,6 +321,31 @@ Reviewer's check list - QA
   functional tests?
 
 
+Reviewer's check list - Support
+-------------------------------
+
+* Does the documentation contain old information that needs to be updated?
+
+* Does the documentation makes sense to new and existing users?
+
+* UX: Does the Local Manager make sense along with the text configuration?
+
+* If it's a new feature release, does the information make sense to new
+  users?
+
+* Should there be an accompanying Users Guide? FAQ update? API addition?
```

Comment:
Not if we can decide this now... usually the user guides are created based on customer feedback.

otherwise this is a duplicate of  `Does the documentation makes sense to new and existing users?` 

userguides faq , API are all part of the documentation